avm/utilities/tools/Invoke-WorkflowsForBranch.ps1

```diff
@@ -88,16 +114,19 @@ Get a list of all GitHub module workflows
 Get a list of all GitHub module workflows. Does not return all properties but only the relevant ones.
 
 .PARAMETER PersonalAccessToken
-Mandatory. The GitHub PAT to leverage when interacting with the GitHub API.
+Optional. The PAT to use to interact with either GitHub / Azure DevOps. If not provided, the script will use the GitHub CLI to authenticate.
 
 .PARAMETER RepositoryOwner
 Mandatory. The repository's organization.
 
 .PARAMETER RepositoryName
 Mandatory. The name of the repository to fetch the workflows from.
 
+.PARAMETER IncludeDisabled
+Optional. Set if you want to also include disabled workflows in the result.
+
 .PARAMETER Filter
-Optional. A filter to apply when fetching the workflows. By default we fetch all module workflows (avm.res.*).
+Optional. A regex filter to apply when fetching the workflows. By default we fetch all module workflows (avm.res.*).
```

Comment:
Either the below or we should update the removed expression to also include ptn and utl
```suggestion
Optional. A regex filter to apply when fetching the workflows. By default we fetch all module workflows.
```